source/localizable/tutorial/service.md

```diff
@@ -0,0 +1,293 @@
+For Super Rentals, we want to add the ability to display maps of where each rental is located.  To implement this feature, we will take advantage of several Ember concepts:
```

Comment:
For Super Rentals, we want to be able to display a map showing where each rental is.